x-pack/solutions/observability/plugins/slo/ui_tests/services/slo_data_service.ts

```diff
@@ -81,8 +69,8 @@ export class SLoDataService {
       groupBy: ['user.id'],
```

Comment:
Add this settings to the created slo payload, it might help with the time it takes to generate the SLO
```
settings: {
 preventInitialBackfill: true
}
```

Reply:
Done